src/normalizations/normalize-message.ts

```diff
@@ -126,7 +126,7 @@ export const normalizeMailMessageFromSoap = (
 			shr: m.shr,
 			body: m.mp ? generateBody(m.mp || [], m.id) : undefined,
 			isComplete,
-			read: !isNil(m.f) ? !/u/.test(m.f) : undefined,
+			read: !isNil(m.f) ? !/u/.test(m.f) : true,
```

Comment:
wouldn't it be safer to apply the same logic also to the other flags?

Reply:
Yes ,it would be better.I'll check the docs for the default value then will update it